packages/ts-types/src/narrowing/as.ts

```diff
@@ -115,16 +115,16 @@ export function asPlainObject<T = object>(value: unknown, defaultValue?: T): Opt
  *
  * @param value The value to test.
  */
-export function asDictionary<T = object>(value: unknown): Optional<Dictionary<T>>;
+export function asDictionary<T = unknown>(value: unknown): Optional<Dictionary<T>>;
```

Comment:
This was a bug but the fix could technically be considered a breaking change.  I highly doubt it's been used by anyone, but who can say for sure...